### PR TITLE
Release config

### PR DESCRIPTION
**Pull request by Fiszcz**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,26 +1,51 @@
...
+        "build": "tsc -p tsconfig.json",
+        "release": "semantic-release"
+    },
+    "dependencies": {
+        "qs": "^6.10.1",
+        "ts-morph": "^10.0.2"
     },
     "devDependencies": {
+        "@semantic-release/changelog": "^5.0.1",
+        "@semantic-release/git": "^9.0.0",
```
Comment (pawfa) [inline package.json:30]: @semantic-release/changelog and @semantic-release/git can be removed, they are not used anywhere.

**Review by pawfa — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (Fiszcz) [reply package.json]: ok, removed

File: .github/workflows/node.js.yml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js 14.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14.x
```
Comment (pawfa) [inline .github/workflows/node.js.yml:19]: Why only 14.x version? I think 15.x would be worth checking too as it is current release.

File: husky.config.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+module.exports = {
+    hooks: {
+        'pre-commit': 'pretty-quick --stagged',
```
Comment (pawfa) [inline husky.config.js:3]: typo stagged -> staged

File: package.json [hunk 4]
```diff
@@ -1,26 +1,49 @@
...
+    "description": "Generating pact files from typescript definitions",
+    "keywords": [
+        "pacts",
+        "contracts",
+        "tests",
+        "typescript"
+    ],
     "main": "./lib/index.js",
+    "bin": {
+        "pact-gen-ts": "./bin/pact-gen-ts.js"
```
Comment (pawfa) [inline package.json:13]: Maybe "pact-gen" as the cli invocation would be enough? Its always a bit less typing :)

File: .github/workflows/node.js.yml (hunk 2 above)
Comment (Fiszcz) [reply .github/workflows/node.js.yml:19]: Yes, sure.

But I think that if I do not have tests yet in my code, it is not important.

And Node.js 15 is odd-number version, so not for production using.
Now Node.js 14  STL is enough for now.

File: husky.config.js (hunk 3 above)
Comment (Fiszcz) [reply husky.config.js:3]: nice catch, thanks

File: package.json (hunk 4 above)
Comment (Fiszcz) [reply package.json:13]: I think that the ending "ts" is very important for command which generating pacts from typescript.

**Pull request merged**